.eslintrc.json

```diff
@@ -0,0 +1,70 @@
+{
+  "extends": [
+    "eslint:recommended",
+    "plugin:node/recommended",
+    "plugin:promise/recommended",
```

Comment:
I'm not sure we use any promise code or async/await... should we keep this in anyway?

Reply:
We need to weigh the cost of having it therr now vs the risk that we forget to add it when it is needed. I'm fine either way but leaning towards *keep*.